source/basic.tex

```diff
@@ -569,22 +569,21 @@
 \end{itemize}
 in a program provided that
 each definition appears in a different translation unit, and
-provided the definitions satisfy the following requirements.
-There shall not be more than one definition
-of an entity
-that is attached to a named module\iref{module.unit};
-no diagnostic is required unless a prior definition
-is reachable at a point where a later definition appears.
-Given such
-an entity named \tcode{D} defined in more than one translation unit,
-all of the following requirements shall be satisfied.
+the definitions satisfy the following requirements.
+Given such an entity \tcode{D} defined in more than one translation unit,
+for all definitions of \tcode{D}, or,
+if \tcode{D} is unnamed enumeration,
```

Comment:
"is unnamed enumeration" -> "is an unnamed enumeration"

Reply:
Fixed.